src/TerraformTaintCommand.groovy

```diff
@@ -0,0 +1,37 @@
+class TerraformTaintCommand implements TerraformCommand, Pluggable<TerraformTaintCommandPlugin> {
+    private String command = "taint"
+    private String resource
+    private String environment
```

Comment:
Is `environment` dead code?

Reply:
Gross, it's part of the `TerraformCommand` interface.

Reply:
Cutting a separate issue for that - https://github.com/manheim/terraform-pipeline/issues/357

Reply:
Yeah, that was part of why I moved environment management into it when I was converting it to a trait - it already had getEnvronment() in the interface.